src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderTestData.java

```diff
@@ -113,7 +115,12 @@ private static SnapshotBuilderProgramDataListOption generateSnapshotBuilderProgr
           .programDataOptions(
               List.of(
                   generateSnapshotBuilderProgramDataRangeOption(
-                      1, "person", "year_of_birth", "Year of birth", 0, 100),
+                      1,
+                      PersonConstants.PERSON,
+                      PersonConstants.YEAR_OF_BIRTH,
+                      "Year of birth",
+                      0,
+                      100),
                   generateSnapshotBuilderProgramDataListOption(
                       2,
                       "person",
```

Comment:
Maybe this and these person columns below could be constants too?

Reply:
Good catch!